download_deps_CN.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+# PEP 723 metadata
+# /// script
+# requires-python = ">=3.10"
+# dependencies = [
+#   "huggingface-hub",
+#   "nltk",
+# ]
+# ///
+
+from huggingface_hub import snapshot_download
+import nltk
+import os
+import urllib.request
+
+urls = [
```

Comment:
Adding a CLI argument to download_deps.py instead of duplicating most code of download_deps.py.